### ISSUE TITLE
PostgreSQL 9.4 Preparation

### ISSUE THREAD

**Issue by alanorth** [ilri/DSpace#183]
We've been using PostgreSQL 9.3 for about two years now, and should be looking to move to 9.4 soon.

[PostgreSQL 9.4 brings performance benefits](http://blog.pgaddict.com/posts/performance-since-postgresql-7-4-to-9-4-fulltext) but also needs some attention to [huge_page settings in its configuration as well as the system's kernel](http://www.postgresql.org/docs/9.4/static/kernel-resources.html).

**Comment by oguya** [comment]
Why not 9.5? It already contains 9.4's performance benefits plus other new features
https://wiki.postgresql.org/wiki/What's_new_in_PostgreSQL_9.5

Besides that, 9.5 is the default version on Ubuntu 16.04 :wink:

**Comment by rradillen** [comment]
incidentally is 9.4 minimum for dspace 6.

**Comment by alanorth** [comment]
@oguya Yah, when I opened this issue four months ago (😉) I mostly just wanted to make a note somewhere that 9.4 had some potential issues to consider on the system side (huge pages). Of course we'll go with PostgreSQL 9.5 when we finally move to the new Ubuntu hotness! Also, when has the "default version on Ubuntu" stopped you from installing a PPA and overriding the distro's version against my advice? :wink:

@rradillen Nice! Well at this rate it will be another year before we get to DSpace 6. I'm still trying to find time to rebase all our crap on top of 5.5 (and we just got the compatible Atmire modules a month or two ago).
